index.html

```diff
@@ -26,5 +26,183 @@ <h1>José Fernández-Caminero</h1>
+            </button>
+        </section>
+
+        <section class="box1">
+            <img src="resources/Rectangle 21(3).png" alt="Picture of laptop and other objects">
+
+            <h3 class="projectName">Project name goes here</h3>
+
+            <ul class="greenul">
+                <li class="greenli">HTML/CSS</li>
+                <li class="greenli">Ruby on Rails</li>
+                <li class="greenli">JavaScript</li>
+            </ul>
+
+            <button class="bordersolid">See this project
+                <img src="resources/Union.png" alt="Cool Arrow Pointing Right">
+            </button>
+        </section>
+    
+        <section class="box1">
+            <img src="resources/Rectangle 21(4).png" alt="Picture of laptop and other objects">
+
+            <h3 class="projectName">Project name goes here</h3>
+
+            <ul class="greenul">
+                <li class="greenli">HTML/CSS</li>
+                <li class="greenli">Ruby on Rails</li>
+                <li class="greenli">JavaScript</li>
+            </ul>
+
+            <button class="bordersolid">See this project
+                <img src="resources/Union.png" alt="Cool Arrow Pointing Right">
+            </button>
+        </section>
+
+        <section class="box1">
+            <img src="resources/Rectangle 21(5).png" alt="Picture of laptop and other objects">
+
+            <h3 class="projectName">Project name goes here</h3>
+
+            <ul class="greenul">
+                <li class="greenli">HTML/CSS</li>
+                <li class="greenli">Ruby on Rails</li>
+                <li class="greenli">JavaScript</li>
+            </ul>
+
+            <button class="bordersolid">See this project
+                <img src="resources/Union.png" alt="Cool Arrow Pointing Right">
+            </button>
+        </section>
+
+        <section class="box1">
+            <img src="resources/Rectangle 21(6).png" alt="Picture of laptop and other objects">
+
+            <h3 class="projectName">Project name goes here</h3>
+
+            <ul class="greenul">
+                <li class="greenli">HTML/CSS</li>
+                <li class="greenli">Ruby on Rails</li>
+                <li class="greenli">JavaScript</li>
+            </ul>
+
+            <button class="bordersolid">See this project
+                <img src="resources/Union.png" alt="Cool Arrow Pointing Right">
+            </button>
+        </section>
+    </section>
+
+    <section>
+        <h1>About</h1>
+
+        <p>
+            I love to code. I think coding is where the arts and the sciences meet. It's the most natural activity for a renaissance person.
+        </p>
+
+        <button>Get my CV
+            <img src="resources/ic_download.svg" alt="Downward Arrow">
+        </button>
+
+        <section class="marginleft">
+            <section class="rectangle">
+                <img src="resources/Rectangle 41.png" alt="Blue Rectangle">
```

Comment:
Kindly, I recommend that you change these background color images and use background colors with CSS.